context/app/static/js/components/detailPage/BulkDataTransfer/BulkDataTransferPanels.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+
+import { useFlaskDataContext } from 'js/components/Contexts';
+
+import BulkDataTransferPanel from './BulkDataTransferPanel';
+import Link from './Link';
+import NoAccess from './NoAccess';
+import { usePanelSet } from './usePanelSet';
+
+function BulkDataTransferPanels() {
+  const {
+    entity: { dbgap_study_url, dbgap_sra_experiment_url },
+  } = useFlaskDataContext();
+
+  const panelsToUse = usePanelSet();
+
+  // Assign dynamic URL's to each type of link
+  const linkTitleUrlMap = {
+    dbGaP: dbgap_study_url,
+    'SRA Experiment': dbgap_sra_experiment_url,
+  };
+
+  if (panelsToUse.error) {
+    return <NoAccess>{panelsToUse.error}</NoAccess>;
+  }
+
+  return (
+    <>
+      {panelsToUse.panels.length > 0 &&
+        panelsToUse.panels.map((props) => <BulkDataTransferPanel {...props} key={props.title} />)}
+      {panelsToUse.links.length > 0 && (
+        <Paper>
+          <>
```

Comment:
We should be able to remove this extra fragment wrapping the `map` since the `Paper` component but it's a non-blocking issue 😄